### DISCUSSION TITLE
Moving some fsprojects repos into fslab

### DISCUSSION THREAD

**Discussion by dsyme** [fslaborg/FsLab#4] [General]
I propose we move these repos from fsprojects to fslab org, without changing owners/admins

https://github.com/fsprojects/ExcelFinancialFunctions

https://github.com/fsprojects/ExcelProvider

These are not super-active projects but I think they will have more affinity in fslab and allow them to be revised/revitalised when and if necessary

What do you think?

**Comment by luisquintanilla** [comment]
That makes sense. Excel is still one of the main ways to inspect and work with data. Considering it's had updates and there is some work being done on extensibility there might be opportunities to leverage add-ins for some of these scenarios

https://docs.microsoft.com/en-us/office/dev/add-ins/excel/excel-add-ins-overview
